src/content/blog/ddev-dec-2024-newsletter.md

```diff
@@ -0,0 +1,42 @@
+
+**DDEV Blogs and News**
+
+* [DDEV Usage Statistics](stats-on-ddev-usage-nov-2024.md)
+* [Setting up a Windows Machine for DDEV](windows-ddev-setup.md)
+* [Contributor Training: DDEV Analytics with Amplitude](https://ddev.com/blog/amplitude-ddev-analytics-contributor-training/)
+* [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). These meetings happen bimonthly and you're always invited. The next one is Wednesday, January 8, 2025, and is the most important meeting of the year. We will review 2024 and plan for 2025. We'd love to have you [join us](https://www.meetup.com/ddev-events/events/303197425).
+
+**DDEV Notes from around the web**
+
+* Drupal lead Dries Buyteart posted [Installing Drupal CMS (or Drupal Starshot) using DDEV](https://dri.es/installing-drupal-cms-or-drupal-starshot-using-ddev).
+* Drupal at your Fingertips contains loads of great DDEV advice, including how to set up Apache Solr, see [Drupal and Solr](https://www.drupalatyourfingertips.com/development#solr-and-ddev) (and lots of advice about other things including Xdebug).
+* [SilverStripe and Elasticsearch with DDEV](https://firesphere.dev/articles/ddev-elasticsearch-and-silverstripe)
+* [Xdebug, DDEV, WSL2 and corporate network restrictions](https://www.koehnlein.eu/en/blog/2024/ddev-wsl-xdebug/)
+* [DDEV matomo add-on](https://www.linkedin.com/pulse/phase-2-week-3-whos-valery-lourie-nskjf/)
+
+
+**Fully funding Maintainer Stas**: *We need your help and your organization's help! Let me know if you need help getting this into your 2025 budget!* Our key financial goal is to [fully fund @stasadev so he can work exclusively on DDEV](lets-fund-stas-maintainer.md). We’re about 30% of the way to that goal, but apparently due to market conditions, have been going backward in recent months. We need about $3700/month in increased pledges from organizations and individuals. See [Full information about supporting DDEV](https://github.com/sponsors/ddev). We’re happy to invoice you, happy to [do a call discussing this](https://cal.com/randyfay/30min), and would love it if you’d include DDEV in your 2025 budgeting process. (Our current status: We receive about $3000-$4000/month, spend about $4000/month. Bank balance is about $9,300, up from $9,000 last month.)
+
+**THANKS to all of you who are supporting DDEV’s path to sustainability** and who have gotten your organizations to do so.
+
+Want to keep up as the month goes along? Follow our [blog](https://ddev.com/blog/), [LinkedIn](https://www.linkedin.com/company/ddev-foundation), [Mastodon](https://fosstodon.org/@ddev), and join us on [Discord](https://discord.gg/5wjP76mBJD).
```

Comment:
```suggestion
Want to keep up as the month goes along? Follow our
* [blog](https://ddev.com/blog/), 
* [LinkedIn](https://www.linkedin.com/company/ddev-foundation), 
* [Mastodon](https://fosstodon.org/@ddev), 
* [BlueSky](https://bsky.app/profile/ddev.bsky.social) 
* and join our community on [Discord](https://discord.gg/5wjP76mBJD).
```

Added BlueSky
Slight formatting might improve brand recognition for visual scanner?
Highlight the community hangs out at Discord

Reply:
Thanks!